examples/multisig_plain/lib.rs

```diff
@@ -532,13 +527,7 @@ mod multisig_plain {
             confirmer: AccountId,
             transaction: TransactionId,
         ) -> ConfirmationStatus {
-            if !self.confirmation_count.contains_key(&transaction) {
-                self.confirmation_count.insert(transaction, 0);
-            }
-            let count = self
-                .confirmation_count
-                .get_mut(&transaction)
-                .expect("just inserted if missing");
+            let count = self.confirmation_count.entry(transaction).or_insert(0);
```

Comment:
This change does not preserve semantics. Before the change we panicked upon not receiving a confirmation count and thus could tell if there was a bug and now we silently add a `0` to the confirmation count hash table which adds "gargabe" data.

Reply:
I think you missed the three lines before `let count = …` which I also replaced:
```
if !self.confirmation_count.contains_key(&transaction) {
    self.confirmation_count.insert(transaction, 0);
}
```
So my change is equivalent and this "garbage 0" was also added prior to it.

Reply:
yep, thanks! sorry